python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -419,12 +421,12 @@ def getMetadataBox(cls, bbox, wcs, filterName=None, calib=None, epoch=None, bbox
         for box, corners in zip(("INNER", "OUTER"), (innerCorners, outerCorners)):
             for (name, corner) in zip(("UPPER_LEFT", "UPPER_RIGHT", "LOWER_LEFT", "LOWER_RIGHT"),
                                       corners):
-                md.add(f"{box}_{name}_RA", corner.getRa().asDegrees(), f"{box}_corner")
-                md.add(f"{box}_{name}_DEC", corner.getDec().asDegrees(), f"{box}_corner")
+                md.add(f"{box}_{name}_RA", geom.SpherePoint(corner).getRa().asDegrees(), f"{box}_corner")
+                md.add(f"{box}_{name}_DEC", geom.SpherePoint(corner).getDec().asDegrees(), f"{box}_corner")
         md.add("SMATCHV", 1, 'SourceMatchVector version number')
         filterName = "UNKNOWN" if filterName is None else str(filterName)
         md.add('FILTER', filterName, 'filter name for photometric data')
-        md.add('EPOCH', "NONE" if epoch is None else epoch, 'Epoch (TAI MJD) for catalog')
+        md.add('EPOCH', "NONE" if epoch is None else str(epoch), 'Epoch (TAI MJD) for catalog')
```

Comment:
I would add a comment here noting that this is a workaround for code that probably never worked, so some future person fixing the underlying problem doesn't think it's sacrosanct.